tardis/conftest.py

```diff
@@ -133,10 +133,10 @@ def pytest_report_header(config):
     if opts:
         s += "Using Astropy options: {0}.\n".format(" ".join(opts))
 
-    if six.PY3 and (config.getini('doctest_rst') or config.option.doctest_rst):
+    if not six.PY2 and (config.getini('doctest_rst') or config.option.doctest_rst):
```

Comment:
I just ran across that and thought, why not fix it. If I run across a potential bug, is there any reason to not fix it?